nss-day-cohort-48/rare-client-spicy-sauce-scouts
# post displaying on user profile

​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jca-post-display
    ```
2. Open a new Terminal tab  and navigate to the server directory.
3. Test app functionality.
    navigate to profile and ensure that posts are showing for that specific user
